lib/ansible/plugins/cliconf/__init__.py

```diff
@@ -55,11 +53,13 @@ class CliconfBase(with_metaclass(ABCMeta, object)):
     """
     A base class for implementing cli connections
 
-    .. note:: String inputs to :meth:`send_command` will be cast to byte strings
-         within this method and as such are not required to be made byte strings
-         beforehand.  Please avoid using literal byte strings (``b'string'``) in
-         :class:`CliConfBase` plugins as this can lead to unexpected errors when
-         running on Python 3
+    .. note:: Unlike most of Ansible, nearly all strings in
```

Comment:
Please do not revert this doc change. The old text is wrong in almost every important way.

Reply:
Fixed